src/dev/impl/DevToys/Helpers/HashingHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
```

Comment:
Please add `#nullable enable` at the top of the file.

Reply:
Missed this one!